openassessment/xblock/static/js/src/lms/oa_staff_info.js

```diff
@@ -58,7 +58,41 @@ OpenAssessment.StaffInfoView.prototype = {
                 view.showLoadError('student_info');
             });
     },
+    
+    /**
+     Upon request, submits an override grade.
+     **/
+    peerScoreOverride: function() {
+    	var sel = $('#openassessment__staff-info', this.element);
+    	var student_id = sel.find('#openassessment__student_id').val();
+    	var override_score = sel.find('#openassessment_override_score').val();
+    	var points_possible = sel.find('#openassessment_points_possible').html();
```

Comment:
override_score comes from the input box, points_possible comes from the html table element.